Mathlib/CategoryTheory/Opposites.lean

```diff
@@ -467,6 +591,82 @@ protected def unop {F G : Cᵒᵖ ⥤ Dᵒᵖ} (α : F ≅ G) : G.unop ≅ F.uno
   hom := NatTrans.unop α.hom
   inv := NatTrans.unop α.inv
 
+@[simp]
+theorem unop_refl (F : Cᵒᵖ ⥤ Dᵒᵖ) : NatIso.unop (Iso.refl F) = Iso.refl F.unop := rfl
+
+@[simp]
+theorem unop_trans {F G H : Cᵒᵖ ⥤ Dᵒᵖ} (α : F ≅ G) (β : G ≅ H) :
+    NatIso.unop (α ≪≫ β) = NatIso.unop β ≪≫ NatIso.unop α :=
+  rfl
+
+@[simp]
+theorem unop_symm {F G : Cᵒᵖ ⥤ Dᵒᵖ} (α : F ≅ G) : NatIso.unop α.symm = (NatIso.unop α).symm := rfl
+
+lemma op_isoWhiskerRight {E : Type*} [Category E] {H : D ⥤ E} (α : F ≅ G) :
+    NatIso.op (isoWhiskerRight α H) =
+    (Functor.opComp _ _) ≪≫ isoWhiskerRight (NatIso.op α) H.op ≪≫ (Functor.opComp _ _).symm := by
+  aesop_cat
+
+lemma op_isoWhiskerLeft {E : Type*} [Category E] {H : E ⥤ C} (α : F ≅ G) :
+    NatIso.op (isoWhiskerLeft H α) =
+    (Functor.opComp _ _) ≪≫ isoWhiskerLeft H.op (NatIso.op α) ≪≫ (Functor.opComp _ _).symm := by
+  aesop_cat
+
+lemma unop_whiskerRight {F G : Cᵒᵖ ⥤ Dᵒᵖ} {E : Type*} [Category E] {H : Dᵒᵖ ⥤ Eᵒᵖ} (α : F ≅ G) :
+    NatIso.unop (isoWhiskerRight α H) =
+    (Functor.unopComp _ _) ≪≫ isoWhiskerRight (NatIso.unop α) H.unop ≪≫
+      (Functor.unopComp _ _).symm := by
+  aesop_cat
+
+lemma unop_whiskerLeft {F G : Cᵒᵖ ⥤ Dᵒᵖ} {E : Type*} [Category E] {H : Eᵒᵖ ⥤ Cᵒᵖ} (α : F ≅ G) :
+    NatIso.unop (isoWhiskerLeft H α) =
+    (Functor.unopComp _ _) ≪≫ isoWhiskerLeft H.unop (NatIso.unop α) ≪≫
+      (Functor.unopComp _ _).symm := by
+  aesop_cat
+
+lemma op_leftUnitor :
+    NatIso.op (Functor.leftUnitor F) =
```

Comment:
For the lemmas involving the unitors, could you use the syntax `F.leftUnitor` instead of `Functor.leftUnitor F`?